packages/tag/src/components/styledComponent/StyledTag.js

```diff
@@ -1,48 +1,76 @@
-import styled from "styled-components";
+import styled, { css } from "styled-components";
 import { colors as colorConstants } from "@crave/farmblocks-theme";
 
+import tagTypes from "../../constants/tagTypes";
+
 const rightPadding = props => (props.onRemove ? "0px" : "8px");
 
+const typeStyles = {
+  [tagTypes.SECONDARY]: css`
+    border-color: ${colorConstants.GREY_16};
+    background-color: ${colorConstants.INDIGO_MILK_CAP};
+    color: white;
+    &:hover {
+      border-color: ${colorConstants.GREY_32};
+      background-color: ${colorConstants.BLUE_CORN};
+    }
+    .icon {
+      color: ${colorConstants.GREY_32};
+    }
+    &:hover .icon,
+    &:focus .icon {
+      color: white;
+    }
+  `,
+  [tagTypes.NEUTRAL]: css`
+    border-color: ${colorConstants.GREY_16};
```

Comment:
border-color is the same on secondary and neutral... maybe we could leave it on the `StyledTag`